policy/release/slsa_source_correlated_test.rego

```diff
@@ -0,0 +1,128 @@
+package policy.release.slsa_source_correlated
+
+import future.keywords.in
+
+import data.lib
+
```

Comment:
Maybe add a success case where no warnings and no violations are reported?

Reply:
The `test_*_happy_day` tests should cover this

Reply:
ack. Missed that.